### PR TITLE
Clarify the meaning of 'const entries'

### PR DESCRIPTION
**Pull request by jafingerhut**

### CODE REVIEW

**Comment by jafingerhut** [comment]
This PR is intended only to clarify the existing practice of P4 compilers and run times regarding their behavior when a table's entries are declared using `const entries` in the source code.  I am not aware of any P4 implementations that differ from the description in these changes.

File: p4-16/spec/P4-16-spec.mdk [hunk 1]
```diff
@@ -4054,7 +4054,7 @@ the `struct` or `header` types do need to be initialized though.
 
 Some P4 expressions denote sets of values (`set<T>`, for some type `T`;
 see Section [#sec-set-types]). These expressions can
-appear only in a few contexts---parsers and constant table
```
Comment (jafingerhut) [inline p4-16/spec/P4-16-spec.mdk:4057]: This and the next small change in this PR are not strictly necessary, but the table property name is technically `entries`, not `const entries`, although currently the `const` is required whenever `entries` appears.  You can also consider these minor changes as prelude to defining new semantics for `entries` without `const`, which I hope to present to the LDWG next Monday (first Monday of October), if there is time available in the meeting's agenda.

Comment (mbudiu-vmw) [inline p4-16/spec/P4-16-spec.mdk:4057]: we may change our mind and allow non-const entries too.

**Comment by mbudiu-vmw** [comment]
You can paste that item in the agenda for next week.

**Review by jnfoster — APPROVED** [review]
These seem like helpful clarifications. I approve.

**Comment by jfingerh** [comment]
@mbudiu-vmw "You can paste that item in the agenda for next week." Sorry for my ignorance, but I don't know where to go to do such pasting.  If you do not mind adding it to the agenda, that would be much appreciated.

**Comment by jnfoster** [comment]
Here:
https://docs.google.com/document/d/1XSgdXeG1UuF1FM_XAqxDrHeN4dHZWBnJPKVS6SnGNwM/edit

**Comment by mbudiu-vmw** [comment]
This link is on the front page of the spec repository.

**Comment by mbudiu-vmw** [comment]
To add a changelog entry

**Review by mbudiu-vmw — APPROVED** [review]
I think we agreed to merge this

**Pull request merged**